ONSdigital/dp-collection-api
# Adds swagger spec

### What

Adds swagger spec for the this API. The `get /collections` should match what the code does. 

The parameters for other endpoints will be in future PRs

### How to review

1. Copy and paste the swagger spec into [editor.swagger.io](https://editor.swagger.io/). Check that no errors pop up
2. Check that what is expected of the `get /collections` enpoint matches the spec

### Who can review

Anyone. Even Carl
